app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/ProfileViewModel.kt

```diff
@@ -43,6 +52,15 @@ class ProfileViewModel @Inject constructor(private val repository: ProfileReposi
   private val _songLists = MutableStateFlow<List<SongList>>(emptyList())
   val songLists: StateFlow<List<SongList>> = _songLists
 
+  private val _spotifySubsectionList = MutableStateFlow<List<ListItem>>(emptyList())
+  val spotifySubsectionList: StateFlow<List<ListItem>> = _spotifySubsectionList
+
+  private val _mainList = MutableStateFlow<List<ListItem>>(emptyList())
```

Comment:
MainLidt seems like bad naming, mainItemList could be better I think, because i dont' know if it's items or tracks etc... I think we can show Spotify how to do good naming.